internal/postprocessor/config.yaml

```diff
@@ -135,110 +135,125 @@ modules:
   - workstations
 
 manual-clients:
-  - distribution-name: cloud.google.com/go/bigquery
+  - api-shortname: bigquery
+    distribution-name: cloud.google.com/go/bigquery
     description: BigQuery
-    language: Go
+    language: go
     client-library-type: manual
-    docs-url: https://cloud.google.com/go/docs/reference/cloud.google.com/go/bigquery/latest
-    release-level: ga
+    client-documentation: https://cloud.google.com/go/docs/reference/cloud.google.com/go/bigquery/latest
```

Comment:
ah, but this change will require a code change in manifest.go to parse this field. That will require a new sha

Reply:
It shouldn't, the field is already there in the `ManifestEntry` type with the appropriate tags. I think this file wasn't changed from `docs-url` to `client-documentation` when the `ManifestEntry` switched to using that property name instead. https://github.com/googleapis/google-cloud-go/blob/fc3d84058b8932152408bc3ee0a5584dfe0b0c19/internal/postprocessor/manifest.go#L39

Reply:
Just ran it locally and included the changes here. Worked great!

Reply:
nice! nvm then!